examples/time_series/noaa_json.py

```diff
@@ -0,0 +1,74 @@
+"""
+===================================================================
+Plotting of near real time (NRT) data by fetching it from json file
+====================================================================
+
+How to query and visualize nrt data(json format) using pandas and plot
+it using TimeSeries Library
+"""
+
+
+from collections import OrderedDict
+
+import pandas as pd
+
+from astropy import units as u
+
+from sunpy import timeseries as ts
+from sunpy.time import parse_time
+
+data = pd.read_json("https://services.swpc.noaa.gov/json/goes/primary/xrays-7-day.json")
+
+############################################
```

Comment:
These should all be 79 or 80 characters long (I forgot which) to match our existing examples.